djangoplicity/newsletters/templates/newsletters/newsletter_detail.html

```diff
@@ -1,4 +1,4 @@
-{% extends 'base.html' %}
+{% extends 'newsletters/base.html' %}
```

Comment:
Why was this necessary?.

Reply:
Sorry, this was causing me an error with the previous test data that I was using, I'll remove it right away

Reply:
Well, I just run the tests after removing it and it is causing me an error
![image](https://user-images.githubusercontent.com/9884128/96051750-ce325400-0e41-11eb-8ad2-72d41f285762.png)

Reply:
Ok, then I believe it's actually a bug.

Reply:
Yeah, I'll leave this here for reference: https://github.com/djangoplicity/djangoplicity-newsletters/runs/1190341999

Reply:
This change caused an bad display of the newsletters for NOIRLab, I'm going to revert it.